.gitmodules

```diff
@@ -35,5 +35,14 @@
 	path = plugins/sqs/aws-sdk-cpp
 	url = https://github.com/hpcc-systems/aws-sdk-cpp
 [submodule "configuration/configmgr/RapidJSON"]
-        path = configuration/configmgr/RapidJSON
-        url = https://github.com/hpcc-systems/rapidjson
+    path = configuration/configmgr/RapidJSON
+    url = https://github.com/hpcc-systems/rapidjson
+[submodule "esp/services/ws_sql/libantlr3c"]
```

Comment:
We will want these to refer to the hpcc-systems fork rather than your personal one.

Reply:
Absolutely.  I just was going to have Ming test it with my repository before I bothered you to make official ones for us.